### ISSUE TITLE
Release ViewFunction declares an opaque return type 'some View'

### ISSUE THREAD

**Issue by paul-brenner** [mergesort/Recap#20]
I'm seeing this error

```
Function declares an opaque return type 'some View', but the return statements in its body do not have matching underlying types
ReleaseView.swift:40
```
when I try to build and archive in XCode cloud with version 1.1.1

I'm not sure what I'm missing and why 1.1.1 is working well for everyone else.

**Comment by mergesort** [comment]
I can't figure out why it's not compiling in some cases, but I was able to fix it in [1.1.2](https://github.com/mergesort/Recap/releases/tag/1.1.2) by swapping a `#guard available` statement for an `#if available` condition. Could you let me know if this fixes the issue you're seeing @paul-brenner?

**Comment by paul-brenner** [comment]
Confirmed that fixed it! Thank you!
